x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_chart_flyout/datafeed_chart_flyout.tsx

```diff
@@ -443,6 +443,14 @@ export const DatafeedChartFlyout: FC<DatafeedChartFlyoutProps> = ({
                             defaultMessage: 'Count',
                           })}
                           position={Position.Left}
+                          domain={
+                            bucketData.length && bucketData[bucketData.length - 1][1] === 0
```

Comment:
I think this has an unwanted side effect where it sets the max to 10 whenever the count of the last bucket is `0`:

<img width="812" alt="image" src="https://user-images.githubusercontent.com/7405507/221215332-affd442d-d6bc-421b-b782-d7e21398550b.png">

Reply:
Good catch, @peteharverson - fixed in 688edfab8440c1da704e67c2b932ca19c247718e